README.md

```diff
@@ -26,11 +25,11 @@ After image publishing, github action will generate kubernetes manifests and pus
 under a directory matching the workflow name. This repo is used to deploy the workflows to an environment
 with [Sonataflow operator][2] running.
 
-## To introduce a new workflow
-1. create a folder under the root with the name of the flow, e.x `/onboarding`
-2. copy `application.properties`, `onboarding.sw.yaml` into that folder
-3. create a github workflow file `.github/workflows/${workflow}.yaml` that will call `main` workflow (see greeting.yaml)
-4. create a pull request but don't merge yet.
+## To introduce a new workflow:
```

Comment:
I don’t have a strong opinion on this, but I prefer not to use a colon at the end of a subsection title.
What about changing the title to 'How to introduce a new workflow' and adding a brief introductory sentence, such as: 'Follow these steps to successfully add a new workflow:'?

Reply:
sounds good to me